step.yml

```diff
@@ -108,6 +108,18 @@ inputs:
 
       If not specified, the default Build Configuration will be used.
 
+- fallback_provisioning_profile_url_list:
```

Comment:
Can we move this input closer to the other auto codesign control inputs.
Like under the Options or Build environment input category.

Reply:
Moved under Options.